ddp-workspace/projects/ddp-sdk/src/lib/services/stackdriverErrorReporter.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable, Inject, ErrorHandler } from '@angular/core';
+import { ConfigurationService } from './configuration.service';
+import {SessionMementoService} from './sessionMemento.service';
```

Comment:
Getting whitespace tslint warning for this line. Please make sure you turn on tslint in your IDE.
(We probably should add it to the builds too?)
```suggestion
import { SessionMementoService } from './sessionMemento.service';
```

Reply:
My bad, thank you.

Moreover, I'm going to create a separate PR with global linter errors/warnings fixes.